assets/templates/kubernetes.container/schema-b/gotext_multiline.tpl

```diff
@@ -9,19 +9,22 @@
 {{- $picktimedate := generate "timedate" }}
 {{- $timehour := generate "timehour" }}
 {{- $faults := generate "faults" }}
-{{- $pct := generate "Percentage" }}
+{{- $pctmem := generate "PercentageMemory" }}
+{{- $pctcpu := generate "PercentageCPU" }}
+{{- $usage_nanoseconds := generate "usage.nanoseconds" -}}
+{{- $usage_nanocores := generate "usage.nanocores" -}}
 {{- $rangeofid := generate "rangeofid" -}}
 {{- $nodeid := div $rangeofid 110 -}}
 {{- $name :=  generate "container.name" }} 
 {  "@timestamp": "{{$picktimedate}}T{{$timehour}}:{{ $resttime._1 }}:{{ $resttime._2 }}:{{ $resttime._3}}",
    "container":{
       "memory":{
-         "usage": {{divf $pct 1000000}}
+         "usage": {{divf $pctmem 100}}
```

Comment:
I see that you generate values for `PercentageMemory` between 0 and 100, and then you divide them by 100

Unless you still need the values in the 1...100 range somewhere else in the template, you can just set the range as something like the following:
```
  - name: PercentageCPU
    range:
      min: 0.0
      max: 1.0
```

Reply:
In terms of performance

> Unless you still need the values in the 1...100 range somewhere else in the template

despite the above, you use multiple times:
```
{{divf $pctmem 100}}
```

It would be better to assign a variable only once, with the generated value divided by 100
Something like the following (the exact syntax is most likely wrong)
```
{{- $pctmem := generate "PercentageMemory" | divf 100 }}
```

Still having a 0.0...1.0 range is the preferred solution if you only need that range. if you need both 0.0...1.0 and 0...100 it's better to use the range that's most used in the template, and anyway assigning the variable for the "second range" directly to the value with the proper operation applied :)

Reply:
I added this in mul of nanocores and nanoseconds
Thanks